honeybadgermpc/progs/jubjub.py

```diff
@@ -198,15 +194,24 @@ def __str__(self):
 
         return self
 
-    async def mul(self, n):
+    def mul(self, n):
         if not isinstance(n, int):
             raise Exception("Can't scale a point by something which isn't an int!")
 
         return self
 
-    async def double(self):
+    def double(self):
         return self
 
+    async def equals(self, other):
+        if not isinstance(other, SharedIdeal):
+            return False
+
+        return self.curve == other.curve
+
+    async def open(self):
```

Comment:
Same here

Reply:
See prior comment, this also is to match style of `Share` objects today.